wp-content/themes/workingnyc/views/components/collection.twig

```diff
@@ -21,7 +21,11 @@
         {% if this.guides %}
         <div class="grid tablet:grid-cols-2 gap-3 auto-rows-[1fr]">
         {% else %}
-        <div class="grid grid-cols-1 tablet:grid-cols-2 gap-3 auto-rows-[1fr]">
+          {% if this.post.post_type == 'job-boards' %}
```

Comment:
Can you add a check for `is_home` as well? And also can you add it back to 

```
{% include [ post.post_type ~ '/' ~ post.singular ~ '.twig'] with {
                  post: post
                } %}
 ```
 
 I don't know when that got deleted